frame/staking/src/lib.rs

```diff
@@ -1425,14 +1556,19 @@ pub mod pallet {
 
 		fn integrity_test() {
 			sp_std::if_std! {
-				sp_io::TestExternalities::new_empty().execute_with(||
+				sp_io::TestExternalities::new_empty().execute_with(|| {
 					assert!(
 						T::SlashDeferDuration::get() < T::BondingDuration::get() || T::BondingDuration::get() == 0,
 						"As per documentation, slash defer duration ({}) should be less than bonding duration ({}).",
 						T::SlashDeferDuration::get(),
 						T::BondingDuration::get(),
-					)
-				);
+					);
+
+					assert!(
```

Comment:
Also one to make sure smallest bag is larger than ED would be good.

Reply:
https://github.com/paritytech/substrate/pull/9081/commits/d5e0c8dedebcea88b9dc792ea3075f8d0ae2ffb1